.gitignore

```diff
@@ -0,0 +1,41 @@
+# Copyright: Chetna (cheatiiit) https://github.com/cheatiiit
```

Comment:
Hi,
Thanks for reviewing. Finda and @rosariorobinson have told about the addition of license to the file. I have made it as per the example given by @kangaroo5383 

@kangaroo5383 : Could you please help in clearing out how is it suppose to be done?